src/app/screens/Game/index.js

```diff
@@ -1,18 +1,89 @@
 import React, { Component } from 'react';
 
 import Board from './components/Board/index.js';
-import style from './styles.css';
+import './styles.css';
+
+function calculateWinner(squares) {
+  const lines = [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]];
+  for (let i = 0; i < lines.length; i += 1) {
+    const [a, b, c] = lines[i];
+    if (squares[a] && squares[a] === squares[b] && squares[a] === squares[c]) {
+      return squares[a];
+    }
+  }
+  return null;
+}
 
 class Game extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
remove constructor and just use state here

Reply:
Done